pyGHDL/libghdl/flags.py

```diff
@@ -33,25 +33,53 @@
 # ============================================================================
 
 from ctypes import c_bool, sizeof
+from enum import unique, IntEnum
+
+from pyTooling.Decorators import export
 
 from pyGHDL.libghdl import libghdl
 
 __all__ = [
     "Flag_Elocations",
     "Verbose",
+    "MB_Comment",
+    "Explicit",
+    "Relaxed",
     "Flag_Elaborate_With_Outdated",
     "Flag_Force_Analysis",
+    "AMS_Vhdl",
     "Flag_Gather_Comments",
 ]
 
 assert sizeof(c_bool) == 1
 
+
+@export
+@unique
+class VhdlStandard(IntEnum):
+    """An enumeration representing libghdl's internal ``Vhdl_Std_Type`` enumeration type."""
+
+    Vhdl_87 = 0  #: VHDL'87
+    Vhdl_93 = 1  #: VHDL'93
+    Vhdl_00 = 2  #: VHDL'2000
+    Vhdl_02 = 3  #: VHDL'2002
+    Vhdl_08 = 4  #: VHDL'2008
+    Vhdl_19 = 5  #: VHDL'2019
+
+
 Flag_Elocations = c_bool.in_dll(libghdl, "flags__flag_elocations")
 
-Verbose = c_bool.in_dll(libghdl, "flags__verbose")
+Verbose = c_bool.in_dll(libghdl, "flags__verbose")  #: Internal boolean flag representing :option:`-v`.
+MB_Comment = c_bool.in_dll(libghdl, "flags__mb_comment")  #: Internal boolean flag representing :option:`--mb-comment`.
+Explicit = c_bool.in_dll(libghdl, "flags__flag_explicit")  #: Internal boolean flag representing :option:`-fexplicit`.
+Relaxed = c_bool.in_dll(
+    libghdl, "flags__flag_relaxed_rules"
+)  #: Internal boolean flag representing :option:`-frelaxed`.
 
 Flag_Elaborate_With_Outdated = c_bool.in_dll(libghdl, "flags__flag_elaborate_with_outdated")
 
 Flag_Force_Analysis = c_bool.in_dll(libghdl, "flags__flag_force_analysis")
 
+AMS_Vhdl = c_bool.in_dll(libghdl, "flags__ams_vhdl")
```

Comment:
If the matching CLI option is called `-ams`, then please accept this documentation update too. GitHub should offer you a button to integrate this line change into the PR without local modifications.

```suggestion
AMS_Vhdl = c_bool.in_dll(libghdl, "flags__ams_vhdl")  #: Internal boolean flag representing :option:`-ams`.
```